docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerJobProperty.java

```diff
@@ -17,6 +19,7 @@
      * Tag on completion (commit).
      */
     public final boolean tagOnCompletion;
+    @CheckForNull
```

Comment:
This annotation is a part of JSR-305 standard, which is being supported by almost all static analysis tools and IDEs.